### PR TITLE
Add figures and tables

### PR DESCRIPTION
**Pull request by Aariq**
NOTE: I haven't tested out a manubot build yet.

This PR re-names and adds figures to 09.figures-tables.md along with examples of how to add in-text cross references.  I also added some notes to the README about reproducibility. 

I took a crack at captions for figures, but since I didn't make the figures I was really just guessing.  I'd appreciate suggestions or edits to the captions.

### CODE REVIEW

File: content/09.figures-tables.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
 
-<!--Here's a ref to that figure (Figure @fig:github).-->
+<!--Reference this figure in text like this (Figure @fig:github-diagram).-->
+
+![A summary of ways GitHub can be used showing the difficulty and degree of collaborativity for each. The vertical axis represents technical difficulty such that activities higher on the scale require knowledge and use of more GitHub features than activities lower on the scale.  Each activity spans a region of the horizontal axis representing who is involved with or benefits from each activity.  For example, storing data and code mainly benefits individuals or a lab group while making data and code citable and reproducible benefit other labs and the greater community as well.](images/scatterblob_1.png){#fig:scatterblob}
```
Comment (Aariq) [inline content/09.figures-tables.md:7]: @LunaSare, I'd love some input on this.  I just guessed!

**Comment by Aariq** [comment]
If anyone remembers who made the github diagram figure (previously figure2.png) can you tag them?

**Comment by robcrystalornelas** [comment]
Maybe that was @colebrookson who make the GitHub diagram? Is that the one you are referring to?

**Comment by colebrookson** [comment]
I was the point person for that figure, yes! what's up? @Aariq are you looking for the caption? It's in pull request 137, but I'll paste here too in case that's what you're looking for: 

A) Multi-faceted components allow for code writing, small data storage, manuscript writing, and project management to all be done in one place. B) Issues, Pull Requests, Discussions, and Projects allow for team members to ask for feedback, suggest fixes, discuss related ideas, and keep track of all the moving parts of a project. C) All collaborators on a project can be a part of a single repository, with varying push privileges and responsibilities. D) CONTRIBUTING.md, LICENCE.md, & README.md files can allow new team members or others wanting to use materials to understand the project components and learn how they can engage with the project and existing team members.

File: content/09.figures-tables.md (hunk 1 above)
Comment (LunaSare) [reply content/09.figures-tables.md:7]: I think the caption looks great already, @Aariq!
Based on previous discussions with @robcrystalornelas and @emmajhudgins , I would add the following:

"A summary of ways GitHub can be used `in Ecology and Evolution,` showing ~~the~~ `technical` difficulty and degree of collaborativity for each. ~~The vertical axis represents technical difficulty such that~~ Activities higher on the vertical axis require `usage` knowledge ~~and use~~ of more GitHub features than activities lower on the scale.  `On the horizontal axis,` each activity spans a region ~~of the horizontal axis~~ representing who is `potentially` involved with or benefits from each activity.  For example, storing data and code mainly benefits individual `researchers` or `members of` a lab group`,` while making data and code citable and reproducible benefit other labs and the `larger` ~~greater~~ community as well.
`Independently of a users knowledge level of GitHub features, there are ways to use GitHub that allow tapping unto one of the most salient benefits of the platform: facilitating and enhancing collaboration.`

**Review by LunaSare — APPROVED** [review]
It all looks good!

I would delete the csv file that has been moved from `R/data/figure_2_data.csv` to `data/figure_2_data.csv`, because it contains old data and we do not really call it to create the scatterblob. 
The most up to date is the one you saved as `data/scatterblob_data_raw.csv`

**Comment by robcrystalornelas** [comment]
Great, thanks for making the updates @Aariq. I'll go ahead and merge if that sounds good to you @Aariq

**Pull request merged**